src/dashboard/utils/template_fetcher.js

```diff
@@ -43,7 +48,9 @@ class TemplateFetcher {
 
   manifestReq = () => {
     return {
-      url: manifestURL,
+      // FIXME: when the beat hierarchy becomes non-beta then we can
+      // just use manifest URL.
+      url: SETTINGS.store.user.beatHierarchy ? betaManifestURL : manifestURL,
```

Comment:
you can also use `SETTINGS.get('user.beatHierarchy')` and it handles null/undefined